content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -14,70 +14,116 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
+dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-Note that the `--prerelease` flag is required for all instrumentation packages
-because they are all are pre-release.
+Now let's add the automatic instrumentation packages for ASP.NET Core. This will
+give us some automatic spans for each HTTP request to our app.
+
+```shell
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
+```
 
-This will also install the `OpenTelemetry` package.
+_Note that as the Semantic Conventions for attribute names are not currently
+stable the instrumentation package is current not in a released state. That
+doesn't mean that the functionality itself is not stable. This means that you
+need to use the `--prerelease` flag, or install a specific version of the
+package_
 
-### Tracing
+This will also install the `OpenTelemetry` core packages.
 
-Next, add [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry) via
-the `System.Diagnostics` API.
+### Setup
 
-Paste the following code into your `Program.cs` file:
+Next, we need to add OpenTelemetry to our Service Collection in `program.cs` so
+that it's listening correctly.
 
 ```csharp
 using System.Diagnostics;
-
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
-// Define some important constants to initialize tracing with
-var serviceName = "MyCompany.MyProduct.MyService";
-var serviceVersion = "1.0.0";
-
 var builder = WebApplication.CreateBuilder(args);
 
-// Configure important OpenTelemetry settings, the console exporter, and instrumentation library
-builder.Services.AddOpenTelemetryTracing(tracerProviderBuilder =>
+// .. other setup
+
+builder.Services.AddOpenTelemetry()
+    .WithTracing(tracerProviderBuilder =>
+        tracerProviderBuilder
+            .AddSource(DiagnosticsConfig.ActivitySource.Name)
+            .ConfigureResource(resource => resource
+                .AddService(DiagnosticsConfig.ServiceName))
+            .AddAspNetCoreInstrumentation()
+            .AddConsoleExporter());
+
+// ... other setup
+
+public static class DiagnosticsConfig
 {
-    tracerProviderBuilder
-        .AddConsoleExporter()
-        .AddSource(serviceName)
-        .SetResourceBuilder(
-            ResourceBuilder.CreateDefault()
-                .AddService(serviceName: serviceName, serviceVersion: serviceVersion))
-        .AddHttpClientInstrumentation()
-        .AddAspNetCoreInstrumentation()
-        .AddSqlClientInstrumentation();
-});
-
-var app = builder.Build();
-
-var MyActivitySource = new ActivitySource(serviceName);
-
-app.MapGet("/hello", () =>
+    public const string ServiceName = "MyService";
+    public static ActivitySource ActivitySource = new ActivitySource(ServiceName);
+}
+```
+
+At this stage, you should be able to run you site, and see a Console output
+similar to this:
+
+<details>
+<summary>View example output</summary>
+
+```
+Activity.TraceId:            54d084eba205a7a39398df4642be8f4a
```

Comment:
it might be good to call out Activity == Span here or in the intro.

Reply:
I can see that, I wonder whether a link to `manual.md` is enough for now.

Reply:
I think we can link here: https://opentelemetry.io/docs/instrumentation/net/manual/#a-note-on-terminology

That section could probably be reworded to include a table that maps out each of the concepts by name, but that can be tackled in a separate PR if you're not feeling like it's your jam to do here.

Reply:
Added a note to this effect